contracts/deploy/Create3.sol

```diff
@@ -41,7 +53,7 @@ library Create3 {
         deployer.deploy(bytecode);
 
         // checking for codehash instead of code length to support contracts that selfdestruct in constructor
-        if (deployed.codehash == bytes32(0)) revert DeployFailed();
+        if (!deployed.isContract()) revert DeployFailed();
```

Comment:
Do we want to keep this sanity check?

Reply:
@deanamiel please remove this line as we already doing the success check here
https://github.com/axelarnetwork/axelar-gmp-sdk-solidity/pull/32/files#diff-e8da8b3175680e091c0f1df471b0fda0ab20b884aa1dea000c4dd2caab59cdebL15-L17

Reply:
Sounds good